pages/PreviewPage/index.js

```diff
@@ -0,0 +1,68 @@
+import { StyledButton } from "@/components/StyledButton";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import Header from "../../components/Header";
+import PreviewPicture from "../../components/PreviewPicture";
+import { saveAs } from "file-saver";
+
+const imageSrc = "https://i.imgur.com/8nUXl6s.jpeg/";
+const imageName = "8nUXl6s.jpeg";
+
+export default function PreviewPage() {
+  const router = useRouter();
+  const imageID = "8nUXl6s";
+  const { slug } = router.query;
+
+  function handleVariations(event) {
+    event.preventDefault();
+    router.push(`../Variations/${imageID}`);
```

Comment:
it is better to work with absolute paths:

```suggestion
    router.push(`/Variations/${imageID}`);
```